widgetssdk/src/main/java/com/glia/widgets/messagecenter/StatefulEditText.kt

```diff
@@ -0,0 +1,134 @@
+package com.glia.widgets.messagecenter
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.annotation.ColorInt
+import androidx.appcompat.widget.AppCompatEditText
+import com.glia.widgets.R
+import com.glia.widgets.view.unifiedui.extensions.*
+import com.glia.widgets.view.unifiedui.theme.base.ColorTheme
+import com.glia.widgets.view.unifiedui.theme.base.LayerTheme
+import com.glia.widgets.view.unifiedui.theme.base.TextInputTheme
+import com.glia.widgets.view.unifiedui.theme.base.TextTheme
+
+class StatefulEditText @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null) :
```

Comment:
NIT personally I prefer dumb views that are being controlled by controller. I think we should have a team discussion on the subject and decide on an approach.

Reply:
What's the benefit of `controller` for this type of `View`? It only does the `View` specific work(state + theme) and there is no business logic.